src/commands/bundle/create.ts

```diff
@@ -16,6 +16,12 @@ export default class BundleCreate extends Command {
 
   public async run(): Promise<unknown> {
     const { args } = await this.parse(BundleCreate);
+
+    const resolvedDir = path.resolve(args.DIR);
+    if (resolvedDir === process.cwd()) {
```

Comment:
fantastic logic checking. this also stops someone from doing something weird like ../<dir>

Reply:
going to add an extra validation for that

Reply:
i tried this locally and it works as it is now!
```
earthfast-cli⚡ ⇒ npm run run -- bundle create bundle ../earthfast-cli

> earthfast-cli@0.7.0 run
> npm run build && ./bin/run bundle create bundle ../armada-cli


> earthfast-cli@0.7.0 build
> rm -rf ./dist && tsc

 ›   Error: Error: Output directory cannot be the current working directory (.)
```